docs/public/deploying-quarkus-app.adoc

```diff
@@ -0,0 +1,84 @@
+== Deploying a Quarkus Application on Kubernetes/Openshift cluster
+
+In this example we will be deploying a https://github.com/odo-devfiles/quarkus-ex[quarkus component] that uses GraalVM and JDK1.8+.
+
+Before you start make sure you have running openshift/kubernetes cluster and do not forget to login if it is openshift cluster.
```

Comment:
This seems better suited for a Note highlighted section. Also
`Before you start, make sure you have a running openshift/kubernetes cluster and do not forget to login, if it is a openshift cluster.`